src/appleseed/renderer/kernel/denoising/denoiser.h

```diff
@@ -43,15 +43,15 @@ class DenoiserOptions
 {
   public:
     float   m_histogram_patch_distance_threshold; //  Histogram patch distance threshold
-    int     m_patch_radius;                       //  Patch has (1 + 2 x m_patchRadius)^2 pixels
-    int     m_search_window_radius;               //  Search windows (for neighbors) spreads across (1 + 2 x m_patchRadius)^2 pixels
+    size_t  m_patch_radius;                       //  Patch has (1 + 2 x m_patchRadius)^2 pixels
```

Comment:
Our convention for end-of-line comments is to start them with a lower case character and no period at the end.